### PR TITLE
Issue/23/object last modified

### PR DESCRIPTION
**Pull request by kahing**

### CODE REVIEW

File: src/main/java/com/bouncestorage/swiftproxy/v1/ContainerResource.java [hunk 1]
```diff
@@ -237,6 +240,12 @@ public Response listContainer(@NotNull @PathParam("container") String container,
     @XmlRootElement(name = "object")
     @XmlType
     static class ObjectEntry {
+        static class SwiftDateSerializer extends DateSerializer {
```
Comment (timuralp) [inline src/main/java/com/bouncestorage/swiftproxy/v1/ContainerResource.java]: Would this be useful outside of the container resource? Do other resources have dates? I think Containers have their creation date.

File: src/test/resources/run-swiftclient-tests.sh [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+$SWIFT post test_container
+$SWIFT stat test_container
+$SWIFT list
+$SWIFT upload test_container README.md
+$SWIFT stat test_container README.md
+$SWIFT list test_container
+$SWIFT list test_container --lh
+
+EXIT_CODE=$?
+exit $?
```
Comment (andrewgaul) [inline src/test/resources/run-swiftclient-tests.sh]: This does not do what you expect:

```bash
$ ls foo
ls: cannot access foo: No such file or directory
$ bar=$?
$ echo $?
0
$ echo $bar
2
```

File: src/main/java/com/bouncestorage/swiftproxy/v1/ContainerResource.java (hunk 1 above)
Comment (kahing) [inline src/main/java/com/bouncestorage/swiftproxy/v1/ContainerResource.java]: dates returned from http header has normal date format (which is not serialized by jackson). container listing (GET account) does not have container creation date in the output

Comment (timuralp) [inline src/main/java/com/bouncestorage/swiftproxy/v1/ContainerResource.java]: Ah, sorry. I was looking at the headers (although that one is actually in Unix epoch time format) -- never mind!

**Comment by kahing** [comment]
waiting for https://github.com/travis-ci/travis-ci/issues/3966 before travis can pass

**Pull request merged**